container/run.sh

```diff
@@ -13,12 +13,14 @@ DATABASE_URL="sqlite:///%kernel.project_dir%/../data/${DATABASE_NAME}"
 
 if [ -f /zusam/config ]; then
   sed -i -e "s|<SECRET>|$(openssl rand -base64 48)|g" \
-    -e "s|<DOMAIN>|${DOMAIN}|g" \
     -e "s|<ALLOW_EMAIL>|${ALLOW_EMAIL}|g" \
     -e "s|<ALLOW_IMAGE_UPLOAD>|${ALLOW_IMAGE_UPLOAD}|g" \
+    -e "s|<ALLOW_MESSAGE_REACTIONS>|${ALLOW_MESSAGE_REACTIONS}|g"\
```

Comment:
I think we also need to update the file `container/config`?

Do we want all environment variables to be configurable via docker? There are many more in `api/config/services.yaml` that get loaded from the config file (I think...) but I don't think would be able to be set via docker.

Reply:
I fixed `container/config`.

Yeah, I think that making everything available is not necessary. I've chosen to add message_reactions and public_links but they seemed sufficient in my opinion.